### ISSUE TITLE
ENOENT: no such file or directory, scandir

### ISSUE THREAD

**Issue by mightyhorst** [jainpranav1/Nand2Tetris-HDL-Visualizer#2]
Command 'Visualize HDL File' resulted in an error (A system error occurred (ENOENT: no such file or directory, scandir))

```
CHIP Or {
    IN a, b;
    OUT out;

    PARTS:
    Nand (a=nota, b=notb, out=out);
    Not (in=a, out=nota);
    Not (in=b, out=notb);
}
```

**Comment by jainpranav1** [comment]
Thanks for letting me know! I will work on getting a fix for this issue soon.

**Comment by jainpranav1** [comment]
![image](https://user-images.githubusercontent.com/64936978/157277981-60d053ff-d561-4f9f-91d3-bcd5bba35259.png)

Interestingly, it works on my Windows computer whenever I use google chrome. May I ask which operating system and browser you are using?

**Comment by mightyhorst** [comment]
Hi there, thanks for prompt reply. 
I am on a Mac (MacOS Monterey v12.0.1) 
and using the latest Google Chrome (Version 99.0.4844.51 (Official Build))
<img width="306" alt="Screen Shot 2022-03-09 at 7 21 36 AM" src="https://user-images.githubusercontent.com/576254/157319524-01e01add-997a-44f5-8713-a1e26d989d36.png">

**Comment by jainpranav1** [comment]
Thanks for the details! I will try to get a fix soon.

**Comment by jainpranav1** [comment]
The issue should now be fixed with version 0.0.5. The issue was that I was not considering that Mac and Windows describe paths differently (one using backslashes, while the other using forward slashes).

**Comment by jainpranav1** [comment]
If the issue is resolved for you, I will close this issue.
